test/extended/networking/bonds.go

```diff
@@ -0,0 +1,117 @@
+package networking
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	kapiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	nadtypes "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-network][Feature:bond]", func() {
+	oc := exutil.NewCLI("bond")
+	f := oc.KubeFramework()
+
+	g.It("pod with bond interface should be created ", func() {
+		namespace := f.Namespace.Name
+		podName1, podName2 := "pod1", "pod2"
+		bondnad1, bondnad2 := "bondnad1", "bondnad2"
+
+		g.By("creating network attachment definitions")
+		o.Expect(
+			createNetworkAttachmentDefinition(
+				oc.AdminConfig(),
+				namespace,
+				"macvlannad",
+				`{"cniVersion":"0.4.0","name":"macvlan-nad","plugins":[{ "type": "macvlan"}]}`),
+		).To(
+			o.Succeed(),
+			"unable to create macvlan network-attachment-definition",
+		)
+
+		o.Expect(
+			createBondNAD(
+				oc.AdminConfig(),
+				namespace,
+				bondnad1,
+				"10.0.0.10/24",
```

Comment:
You can't assume any particular real IPs are available for use by the test case. This might overlap the cluster network or node network.

We may need to add something to the test config to give some networks that can be used for multus tests. For now in #27449 I suggested using the "reserved for documentation/examples" IP ranges.

Reply:
Done

Reply:
Done
Using "192.0.2.0/24